src/components/form/validation.js

```diff
@@ -19,20 +19,21 @@ const Validation = {
     return value && counter < 1 ? undefined : msg;
   },
   validGithub: (value) => {
-    const re = /https:\/\/github\.com\/[A-z0-9_-]+\/?/;
+    const re = /https:\/\/(www\.)?github\.com\/[A-z0-9_-]+\/?/;
     return re.exec(value) ? undefined : 'valid Github url: https://github.com/username';
   },
   validTwitter: (value) => {
-    const re = /https:\/\/twitter\.com\/[A-z0-9_]+\/?/;
+    const re = /https:\/\/(www\.)?twitter\.com\/[A-z0-9_]+\/?/;
     return re.exec(value) ? undefined : 'valid Twitter url: https://twitter.com/username';
   },
   validLinkedIn: (value) => {
-    const re = /https:\/\/www\.linkedin\.com\/in\/[A-z0-9_-]+\/?/;
+    const re = /https:\/\/(www\.)?linkedin\.com\/in\/[A-z0-9_-]+\/?/;
     return re.exec(value) ? undefined : 'valid LinkedIn url: https://www.linkedin.com/in/username';
   },
   validEmail: (value) => {
     // found somewhere online
-    const re = /^([a-zA-Z0-9_\-.]+)@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.)|(([a-zA-Z0-9-]+\.)+))([a-zA-Z]{2,4}|[0-9]{1,3})(\]?)$/;
+    // eslint-disable-next-line
+    const re = /^((([a-z]|\d|[!#\$%&'\*\+\-\/=\?\^_`{\|}~]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])+(\.([a-z]|\d|[!#\$%&'\*\+\-\/=\?\^_`{\|}~]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])+)*)|((\x22)((((\x20|\x09)*(\x0d\x0a))?(\x20|\x09)+)?(([\x01-\x08\x0b\x0c\x0e-\x1f\x7f]|\x21|[\x23-\x5b]|[\x5d-\x7e]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(\\([\x01-\x09\x0b\x0c\x0d-\x7f]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF]))))*(((\x20|\x09)*(\x0d\x0a))?(\x20|\x09)+)?(\x22)))@((([a-z]|\d|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(([a-z]|\d|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])([a-z]|\d|-|\.|_|~|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])*([a-z]|\d|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])))\.)+(([a-z]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(([a-z]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])([a-z]|\d|-|\.|_|~|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])*([a-z]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])))\.?$/;
```

Comment:
this works, maybe we can integrate later on an email verification so we can remove all of these tests too

Reply:
how do you mean?

Reply:
I found that the easiest way to test if an email is valid is to actually ask the user to validate their email address.

Reply:
Hmmm we don’t have that service yet, to dispatch emails, let’s keep it for now, then once we have the email server up we can remove it